homeassistant/components/bang_olufsen/entity.py

```diff
@@ -54,15 +53,13 @@ class BangOlufsenEntity(Entity, BangOlufsenBase):
     """Base Entity for BangOlufsen entities."""
```

Comment:
Should imports still be this order?

Reply:
This one doesn't matter as the `BangOlufsenBase` doesn't override anything from the Entity